invenio_pages/config.py

```diff
@@ -52,3 +54,14 @@
 
 PAGES_FACETS = {}
 """Available facets defined for this module."""
+
+
+PAGES_ALLOWED_EXTRA_HTML_TAGS = ALLOWED_HTML_TAGS + ["img", "button"]
```

Comment:
Unfortunately `ALLOWED_HTML_TAGS` will use the default here, and not what the sys admin has set in the InvenioRDM instance config. See suggestions below.

```suggestion
PAGES_ALLOWED_EXTRA_HTML_TAGS = ["img", "button"]
```